Documentation/project-docs/clr-configuration-knobs.md

```diff
@@ -9,6 +9,7 @@ Name | Description | Type
 `System.GC.Concurrent` | Enable concurrent GC | boolean
 `System.GC.Server` | Enable server GC | boolean
 `System.GC.RetainVM` | Put segments that should be deleted on a standby list for future use instead of releasing them back to the OS | boolean
+`System.Runtime.TieredCompilation` | Enable tiered compilation | boolean
```

Comment:
This change should go to the script used to generate file otherwise it will be overwritten next time script will be run. Perhpas it should be noted in md file header that all of the file is auto generated and changes should be introduced via clr-configuration-knobs.csx

https://github.com/noahfalk/coreclr/blob/06c5e71f155ab0bfdb815d33c683b28926135fae/Documentation/project-docs/clr-configuration-knobs.csx#L345